x-pack/plugins/graph/public/state_management/url_templates.ts

```diff
@@ -6,10 +6,10 @@
 
 import actionCreatorFactory from 'typescript-fsa';
 import { reducerWithInitialState } from 'typescript-fsa-reducers/dist';
-import { KibanaParsedUrl } from 'ui/url/kibana_parsed_url';
```

Comment:
Since we can't refer to legacy code from NP, for now I just copied the shortened version of `KibanaParsedUrl`. Maybe there is similar service in NP but I didn't find anything similar.